packages/client/wallets/smart-wallet/src/SmartWalletSDK.ts

```diff
@@ -36,10 +40,18 @@ export class SmartWalletSDK {
 
         const crossmintService = new CrossmintWalletService(clientApiKey);
         const errorProcessor = new ErrorProcessor(new DatadogProvider());
-        return new SmartWalletSDK(
-            new SmartWalletService(crossmintService, new ClientDecorator(errorProcessor)),
-            errorProcessor
+        const accountCreator = new AccountCreator(
+            new EOACreationStrategy(),
+            new PasskeyCreationStrategy(crossmintService.getPasskeyServerUrl(), clientApiKey)
         );
+        const smartWalletService = new SmartWalletService(
+            crossmintService,
+            new AccountConfigFacade(crossmintService),
+            accountCreator,
+            new ClientDecorator(errorProcessor)
+        );
```

Comment:
Pulling all instantiation up to here while it's still manageable, it let's us inject more cleanly (ex. passing API key in directly to passkeyCreationStrategy, etc)